docs/templates_system/containers_and_widgets.rst

```diff
@@ -14,7 +14,7 @@ Every container can have default parameters and content, and can be hidden when
 The container twig tag keeps HTML syntax always up to date with JavaScript live management expectations.
```

Comment:
This library has been moved to the components now so the documentation should be moved as well, move it under the components directory, check MultiTenancy component how its organized there